package.json

```
@@ -62,6 +62,7 @@
   "devDependencies": {
     "@babel/core": "^7.14.3",
     "@babel/plugin-transform-react-jsx": "^7.14.3",
+    "@bpmn-io/element-template-chooser": "0.0.5",
```

Nice 🚀  Finally we can _really_ test templates in the properties panel test setup

---

Will leave this suggestion here so I don't forget :sweat_smile: 
```suggestion
    "@bpmn-io/element-template-chooser": "^0.0.5",
```